cpp/ql/src/localReferences.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Jump-to-definition links
+ * @description Generates use-definition pairs that provide the data
+ *              for jump-to-definition in the code viewer.
+ * @kind definitions
+ * @id cpp/jump-to-definition
```

Comment:
We can't have two queries with the same id. But isn't this query identical to `localDefinitions.ql`?

If there must be only one query of each kind/tag/whatever per language, why not use the `@id` to identify it? Actually, what we've done with other queries like this is to use `@kind`. That's how the current `definitions.ql` works and how other special queries like https://github.com/Semmle/ql/blob/master/cpp/ql/src/filters/ClassifyFiles.ql works. So why make a new mechanism for what seems to be a similar effect?

Reply:
Oops, the `@id` being a duplicate is an oversight and I agree should be fixed.

@alexet, can you elaborate on why you thought `@kind definitions` was also appropriate/necessary for this query, as opposed to introducing a new `@kind` for ide integration?

Reply:
It's not identical - this query selects results where the *definition* is in the current file, `localDefinitions.ql` selects results where the *reference* is in the current file.